autopush/router/simple.py

```diff
@@ -137,9 +145,27 @@ def route_notification(self, notification, uaid_data):
             returnValue(RouterResponse(response_body="Delivered"))
         else:
             self.metrics.increment("router.broadcast.miss")
+            if self.udp is not None:
+                yield deferToThread(self._send_udp_wake,
+                                    self.udp)
             returnValue(RouterResponse(202, "Notification Stored"))
 
-    #############################################################
+    def _send_udp_wake(self, udp_info):
+        host = udp_info.get("wakeup_host").get("ip")
+        port = udp_info.get("wakeup_host").get("port")
+        data = json.dumps(udp_info.get("mobilenetwork", {}))
+        if port is not None:
+            host = "%s:%d" % (host, port)
+        response = requests.post(
```

Comment:
This isn't quite right, but close!

* The `POST` request is made to the wake-up server (`https://wakeup.tefdigital.com/wakeup/v1`, IIRC), not the device itself. There's a staging server, too; we can make the URL a config option.
* The body contains `mcc`, `mnc`, `netid`, `port`, and `ip`. We should probably just normalize the data before storing it in Dynamo.
* The body needs to be URL-encoded, not JSON. I think you can just pass `data` as-is to `requests.post`; it'll encode it automatically.